Shared/Supporting Files/Extensions/View+Sheet.swift

```diff
@@ -26,6 +26,9 @@ extension View {
     ///   - idealHeight: The ideal height of the popover.
     ///   - onDismiss: A closure to execute when dismissing the sheet.
     ///   - content: A closure returning the content of the sheet.
+    ///
+    /// > Note: This modifier may not work with other UIKit derived views, such as an `Alert`.
+    ///  Using them both in the same view may cause undefined behavior.
```

Comment:
```suggestion
    /// - Note: This modifier can have conflict with modal presentation views, such as an alert.
    /// When the sheet is presented, it may cause the "already presenting" problem.
```

- Let's use the recommended formatting in Apple [Markup](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/Note.html#//apple_ref/doc/uid/TP40016497-CH39-SW1) reference.
- [`Alert`](https://developer.apple.com/documentation/swiftui/alert) is deprecated. I rephrased the comment a bit.